lib/libzfs_core/libzfs_core.c

```diff
@@ -96,6 +97,18 @@ static int g_fd = -1;
 static pthread_mutex_t g_lock = PTHREAD_MUTEX_INITIALIZER;
 static int g_refcount;
 
+void* __do_send_output(void* voidargs);
```

Comment:
this still has external linkage; is there any reason you can't (drop the declaration and) mark it static?

Reply:
Absolutely no reason I'm aware of. So I've done it and pushed.

Thanks!